cypress/integration/learn/challenge-hot-keys.js

```diff
@@ -0,0 +1,84 @@
+const selectors = {
+  instructions: '.challenge-instructions',
+  instructionsPanel: '.instructions-panel',
+  editorContainer: '.monaco-editor',
+  console: '.output-text'
+};
+
+const links = {
+  classic1:
+    '/learn/responsive-web-design/basic-html-and-html5/say-hello-to-html-elements',
+  classic2:
+    '/learn/responsive-web-design/basic-html-and-html5/headline-with-the-h2-element',
+  frontEnd1:
+    '/learn/responsive-web-design/responsive-web-design-projects/build-a-tribute-page',
+  frontEnd2:
+    '/learn/responsive-web-design/responsive-web-design-projects/build-a-survey-form',
+  backEnd1:
+    '/learn/back-end-development-and-apis/back-end-development-and-apis-projects/timestamp-microservice',
+  backEnd2:
+    'learn/back-end-development-and-apis/back-end-development-and-apis-projects/request-header-parser-microservice',
+  video1:
+    '/learn/scientific-computing-with-python/python-for-everybody/introduction-why-program',
+  video2:
+    'learn/scientific-computing-with-python/python-for-everybody/introduction-hardware-architecture'
+};
+
+describe('The hotkeys should work correctly', () => {
+  beforeEach(() => {
+    cy.visit(links.classic1);
+  });
+
+  it('should be possible to navigate to the next challenge/projects and previous', () => {
+    cy.focused().type('{esc}');
+    cy.focused().type('n');
+    cy.url().should('include', links.classic2);
+    cy.focused().type('p');
+    cy.url().should('include', links.classic1);
+    cy.visit(links.frontEnd1);
+    cy.focused().type('{esc}').type('n');
+    cy.url().should('include', links.frontEnd2);
+    cy.focused().type('p');
+    cy.url().should('include', links.frontEnd1);
+  });
+
+  it('should be possible to navigate to the next (and previous) video', () => {
+    cy.visit(links.videoLink1);
```

Comment:
```suggestion
    cy.visit(links.video1);
```
Small correction

Reply:
Thanks, @gikf sorry for the careless change.